src/vehicle.h

```diff
@@ -1100,7 +1091,6 @@ class vehicle : public JsonSerializer, public JsonDeserializer
     std::map<point, std::vector<int> > relative_parts;    // parts_at_relative(x,y) is used alot (to put it mildly)
     std::set<label> labels;            // stores labels
     std::vector<int> alternators;      // List of alternator indices
-    std::vector<int> fuel;             // List of fuel tank indices
```

Comment:
They don't actually improve performance but add a lot of complexity. Once we have cleaner code I'm going to try for more targeted optimization (probably a calculated type field in `vehicle_part`)